Samples/LineGraph/README.md

```diff
@@ -37,7 +44,7 @@ Our line-graph renderer should take a `DataSeries`, and display the data in a ni
 		</c:PlotData>
 	</Curve>
 
-Notice how the `PlotData` element is inside a `Curve` element. This is because we want to repeat PlotCurvePoints inside a curve, in order to build its shape. As `PlotPoint` doesn't affect a `Curve`, it is safe to place that in the same location, which is why we also place the dot here.
+Notice how the `PlotData` element is inside a `Curve` element. This is because we want to place PlotCurvePoints on a curve, in order to build its shape. Due to `PlotData` being what provides the data, this also means we have to place `PlotPoint` inside the `Curve`. This is safe however, as `PlotPoint` doesn't interfere with the layout of `Curve`.
```

Comment:
might be worth mentioning that we could have just used two instances of PlotData, although it doesn't make a difference here.